model/Dockerfile

```diff
@@ -14,23 +14,30 @@
 # limitations under the License.
 #
 
-FROM python:3.8.0-alpine
+FROM python:3.8.0-slim
 
 RUN mkdir -p /usr/src/app
-
-RUN apk update \
- && apk add jq \
- && rm -rf /var/cache/apk/*
+RUN apt update
+RUN apt install -y jq \
+    wget \
+    2to3
```

Comment:
Why do we need a package translation from python2 to python3?

Reply:
In `run.sh` we uninstall this package. But in general, when we compile the `service.proto` file with `protoc`. those files have issues with importing each other (it wasn't fixed by adding __init__.py). The only solution to fix it was to change the import section to something like:
```
from . import service_pb2
```
So we need to add `from .` to imports. The easiest way that I found was using this package.

Reply:
Maybe in the future we can fully update the code to python 3...